WordPress/Classes/ViewRelated/Jetpack/Install/ViewModel/WPComJetpackRemoteInstallViewModel.swift

```diff
@@ -84,6 +89,16 @@ extension WPComJetpackRemoteInstallViewModel: JetpackRemoteInstallViewModel {
             break
         }
     }
+
+    /// NOTE: There's no guarantee that the plugin installation will be properly cancelled.
+    /// We *might* be able to cancel if the request hasn't been fired; but if it has, it'll probably succeed.
+    ///
+    /// An alternative would be to have a listener that checks if installation completes after cancellation,
+    /// and fires background request to uninstall the plugin. But this will not be implemented now.
```

Comment:
I'm inclined to say that we just disable the ability to cancel when the request is already in progress. It'd make things easier from a code perspective and it'd be more clear for a user. We don't have to make that change now but just something to think about.